spark/src/main/scala/geotrellis/spark/io/avro/AvroEncoder.scala

```diff
@@ -38,14 +38,14 @@ object AvroEncoder {
     baos.toByteArray
   }
 
-  def  decompress(bytes: Array[Byte]): Array[Byte] = {
```

Comment:
Actually, there was no extra space ... it was being removed there, not added.  Sorry about that.